renovate.json

```diff
@@ -9,10 +9,22 @@
   "customManagers": [
     {
       "customType": "regex",
-      "fileMatch": ["^.github/test_dependencies.yaml$"],
+      "fileMatch": [
+        "^.github/test_dependencies.yaml$"
+      ],
       "matchStrings": [
         "#\\s+renovate:\\s+datasource=(?<datasource>.*)\\s+depName=(?<depName>.*)\\s+versioning=(?<versioning>.*)\\n.+'(?<currentValue>.*)'"
       ]
+    },
+    {
+      "customType": "regex",
+      "fileMatch": [
+        "^test/consts/helm.go$"
```

Comment:
Ah, I missed this has to be updated as well.

Reply:
👍 Fixed.